conda/recipes/rapids/meta.yaml

```diff
@@ -69,23 +61,11 @@ requirements:
     - conda-forge::ucx-proc=*=gpu
     - conda-forge::ucx {{ ucx_version }}
 
-test:              # [linux64]
-  imports:         # [linux64]
-    - cucim        # [linux64]
-    - cudf         # [linux64]
-    - cudf_kafka   # [linux64]
-    - cugraph      # [linux64]
-    - cuml         # [linux64]
-    {% if cuda_major == "11" %}
-    - cusignal     # [linux64]
-    {% endif %}
-    - cuspatial    # [linux64]
-    - custreamz    # [linux64]
-    - cuxfilter    # [linux64]
-    - dask_cuda    # [linux64]
-    - dask_cudf    # [linux64]
-    - pylibcugraph # [linux64]
-    - rmm          # [linux64]
```

Comment:
We could run these through [`pkgutil.find_loader`]( https://docs.python.org/3/library/pkgutil.html#pkgutil.find_loader ) in a [`run_test.py` script in the recipe]( https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#run-test-script )

This would let us test for their existence without needing to `import` them (and thus not need a GPU to test)

Reply:
I don't think this is necessary for this PR, maybe file an issue or PR with this proposal later on. I feel comfortable with the current level of testing, which is higher than what we had before.